Documentation/Security/GuidelinesIntegrators/Typoscript.rst

```diff
@@ -41,7 +41,7 @@ The following code snippet gives an example:
      table = tt_content
      select {
        pidInList = 123
-       where = deleted=0 AND uid=###PAGEID###
+       where = deleted=0 AND pid=###PAGEID###
```

Comment:
Since this doesn't make much sense given that `pidInList` is already used, maybe we should continue to use `uid` but rename `PAGEID` to `CONTENTID`. Or pick a different `tt_content` field altogether since the next question here would be why we don't use `uidInList`.

Reply:
I have changed it to use CONTENTID and the parameter fooid. The pidInList could help to avoid to display the contents of the rest of the pages.

Reply:
I guess this will do for now.